### PR TITLE
readthedocs

### PR DESCRIPTION
**Pull request by mlathara**
Thanks to @hillsd for the initial legwork here. You can take a look at it here: https://genomicsdb.readthedocs.io/en/docs/

I did a little cleanup from where Scott left it. Mainly:
- Adding an example Python notebook (needs to be updated once the latest python API changes with protobuf are published)
- Add generated Java, Python, C++ and protobuf docs
- Some cleanup

Definitely could use content review for the docs. And we'll want to update the documentation in the source for some of this, but I think that should be a task for later.

### CODE REVIEW

File: docs/index.rst [hunk 1]
```diff
@@ -0,0 +1,44 @@
+.. GenomicsDB documentation master file, created by
+   sphinx-quickstart on Wed Jun 10 21:36:23 2020.
+   You can adapt this file completely to your liking, but it should at least
+   contain the root `toctree` directive.
+
```
Comment (webberlh) [inline docs/index.rst:5]: Could we add a title block to the top of the index? Currently the landing page shows up as `<no title> -- GenomicsDB` in the browser tab.

Comment (mlathara) [reply docs/index.rst:5]: Done - in my defense, I have too many tabs open to ever see the title for my browser tabs 😆

**Review by webberlh — APPROVED** [review]
Looks good to me!

**Review by nalinigans — APPROVED** [review]
Looks good overall. We can get this refined as we go along, especially the genomicsdb api docs which are not very doxygen friendly to begin with.

**Pull request merged**